libs/ledger-live-common/src/families/casper/bridge/account.ts

```diff
@@ -104,6 +110,9 @@ const getTransactionStatus = async (
   else if (!validateAddress(address).isValid)
     errors.sender = new InvalidAddress();
 
+  if (t.transferId && !validateTransferId(t.transferId).isValid)
```

Comment:
Have we added a test case for this on the integration test suite?

Reply:
Missed it, can add...

Reply:
Done

Reply:
Perfect!